src/workloads/query/TimeseriesTSBSOptimizations.yml

```diff
@@ -0,0 +1,282 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-integration"
+Description: |
+  This workload runs different time-series optimizations with the tsbs dataset. During development,
+  we targeted optimizations that are most relevant to pushdown time-series queries to SBE. Further
+  optimizations should be added to this workload. The data is preloaded by the dsi configuration.
+  See 'configurations/test_control/test_control.tsbs_query_genny.yml' for all  the details.
+  
+  Below are helpful statistics about the collection:
+  1. There are 20736000 documents and 161448 buckets in the collection.
+  2. "tags" is the metaField and "time" is the timeField.
+  3. "tags.hostname" has 100 values that are uniformly distributed.
+  4. The 50th percentile of the number of documents per bucket is 93.
+  5. The 80th percentile of the number of document per bucket is 274.96.
+  6. 40435 buckets are version 1 and 121013 buckets are version 2.
+  7. Predicates on a single measurement value, such as {usage_nice: 5} will hit around 18% of buckets.
```

Comment:
Remove this point (because it seems that the distributions of values for different "usage_*" metrics aren't the same and some of them aren't uniform)?